spec/models/case/base_spec.rb

```diff
@@ -1452,4 +1453,44 @@ def clear_case(kase, t, new_state, acting_team)
       end
     end
   end
+
+  describe '#mark_as_clean!' do
+    it 'unsets the dirty flag' do
+      kase = create :case
+      expect(kase).to be_dirty
+      kase.mark_as_clean!
+      expect(kase).not_to be_dirty
+    end
+  end
+
+
+  describe '#trigger_reindexing' do
+    context 'creating a new record' do
+      it 'sets the dirty flag' do
+        kase = build :case
+        expect(kase).not_to be_dirty
+        kase.save!
+        expect(kase).to be_dirty
+      end
```

Comment:
Shouldn't we check that it enqueues a job in all the specs here?

Reply:
done